### PR TITLE
Feat/2615 add/remove participants to set

### PR DESCRIPTION
**Pull request by adipaul1981**
image

### CODE REVIEW

**Comment by adipaul1981** [comment]
@denis-crsj please test all the Sets features (delete/create/edit tag/add/remove). Thanks!

File: src/components/CohortBuilder/ParticipantsTableView/ParticipantsTable.js [hunk 1]
```diff
@@ -16,11 +16,13 @@ import { SORTABLE_FIELDS_MAPPING } from './queries';
 import FileIcon from 'icons/FileIcon';
 import { MONDOLink } from '../../Utils/DiagnosisAndPhenotypeLinks';
 import SaveSetModal from './SaveSetModal';
+import AddRemoveSaveSetModal from './AddRemoveSaveSetModal';
 import './ParticipantTableView.css';
 import { isPartOfGroup } from 'common/profile';
 import DownloadButton from './DownloadButton';
 import ParticipantSetDropdown from './ParticipantSetDropdown';
 import { SaveSetActionsTypes } from 'store/saveSetTypes';
+import { SetSubActionTypes } from '../../../store/saveSetTypes';
```
Comment (evans-g-crsj) [inline src/components/CohortBuilder/ParticipantsTableView/ParticipantsTable.js:25]: ../.../

File: src/components/CohortBuilder/ParticipantsTableView/SaveSetModal.tsx [hunk 2]
```diff
@@ -178,22 +182,24 @@ const SaveSetModal: FunctionComponent<Props> = (props) => {
   };
 
   const { isLoading, error } = create;
-  const isSaveButtonDisabled = () => error != null;
+  const isSaveButtonDisabled = () => error != null || hasError;
```
Comment (evans-g-crsj) [inline src/components/CohortBuilder/ParticipantsTableView/SaveSetModal.tsx:185]: !error || hasError

File: src/store/saveSetTypes.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { ThunkDispatch } from 'redux-thunk';
 import { Sqon } from './sqon';
 import { RootState } from './rootState';
-import { SaveSetInfo } from '../components/UserDashboard/ParticipantSets';
+import { SetInfo } from '../components/UserDashboard/ParticipantSets';
```
Comment (evans-g-crsj) [inline src/store/saveSetTypes.tsx:4]: ../ not sure it s needed

**Review by evans-g-crsj — APPROVED** [review]

File: src/components/CohortBuilder/ParticipantsTableView/SaveSetModal.tsx (hunk 2 above)
Comment (adipaul1981) [reply src/components/CohortBuilder/ParticipantsTableView/SaveSetModal.tsx:185]: no

**Pull request merged**